components/cluster/pcs/pcs.p5m

```diff
@@ -50,13 +52,23 @@ file path=usr/lib/pcsd/pcsd.conf
 file path=usr/lib/pcsd/pcsd.logrotate
 file path=usr/lib/pcsd/pcsd.pam
 file path=usr/lib/pcsd/pcsd.rb
-file path=usr/lib/pcsd/pcsd.service
-file path=usr/lib/pcsd/pcsd.service-runner
+file path=usr/lib/pcsd/pcsd_action_command.rb
+file path=usr/lib/pcsd/pcsd_exchange_format.rb
+file path=usr/lib/pcsd/pcsd_file.rb
+file path=usr/lib/pcsd/pcsd_remove_file.rb
 file path=usr/lib/pcsd/permissions.rb
-file path=usr/lib/pcsd/public/css/LiberationSans-Bold.ttf
-file path=usr/lib/pcsd/public/css/LiberationSans-BoldItalic.ttf
-file path=usr/lib/pcsd/public/css/LiberationSans-Italic.ttf
-file path=usr/lib/pcsd/public/css/LiberationSans-Regular.ttf
+link path=usr/lib/pcsd/public/css/LiberationSans-Bold.ttf \
+    target=/usr/share/fonts/TrueType/liberation/LiberationSans-Bold.ttf
```

Comment:
Targets are wrong!!!!

Reply:
translates wrong to: use relative paths in links?
Or what do you mean?
The fonts must be linked into the folder with the css files.